qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -1641,10 +1641,7 @@ void FitPropertyBrowser::finishHandle(const Mantid::API::IAlgorithm *alg) {
 
   IFunction_sptr function = alg->getProperty("Function");
   updateBrowserFromFitResults(function);
-  if (!isWorkspaceAGroup() && alg->existsProperty("OutputWorkspace")) {
```

Comment:
This looks like it might still be needed because `!isWorkspaceAGroup()` is always true. So maybe we need to keep `if(alg->existsProperty("OutputWorkspace"))`?